packages/ag-charts-community/src/chart/cartesianChart.ts

```diff
@@ -56,6 +56,7 @@ export class CartesianChart extends Chart {
                 paddedRect: seriesPaddedRect,
                 visible: visibility.series,
                 shouldFlipXY: this.shouldFlipXY(),
+                minRect: (this.series[0] as CartesianSeries<any, any>).getMinRect?.(),
```

Comment:
This could be improved if we passed the series type into `Chart<CartesianSeries>` perhaps?